feat: improved overall look of the series season sidebar

antoniosarro opened the pull request:
The series season sidebar has been enhanced to provide users with information on the number of episodes and the current status of each season.

|![image](https://github.com/user-attachments/assets/153b369e-8cc6-412d-ae1c-f218339453d4)|![image](https://github.com/user-attachments/assets/d14a339b-70c6-4397-8b3b-e45f46ade125)|
|:-:|:-:|
|Dark|Light|

Related to issue #765

antoniosarro committed:
feat: improved overall look of the series season sidebar

ToRoCu commented:
@antoniosarro,  this looks very nice, thanks for the effort and the work on it.

IRHM commented:
Oh wow @antoniosarro that looks beautiful. Sorry this flew under my radar, hoping to merge later today when I have a chance to try it out for v2.0.1.

IRHM committed:
SeasonsList: Responsivity fixes

- Fix responsivity for seasons when names are very long (eg on shows that have custom season names)
- Increase status icon size
- Create color variables for season episodes text for resuability and to fix bug causing z index rendering issues with opacity usage.

IRHM committed:
SeasonsList: checkSeasonStatus: Improve log

IRHM committed:
SeasonsList: Remove $inspect: Causing errors in console for some reason

IRHM commented:
Thanks again @antoniosarro, I made a few changes to the markup/styling to help with responsiveness (specifically for shows with custom season names like `/tv/2328`, the changes make the text overflowing a bit nicer).

IRHM committed:
vars: Fix --text-color-accent for dark theme
